daemons/based/pacemaker-based.h

```diff
@@ -43,10 +43,20 @@ enum cib_client_flags {
     cib_is_daemon      = (UINT64_C(1) << 12),
 };
 
+/*!
+ * \internal
+ * \enum cib_op_attr
+ * \brief Bit flags for CIB operation attributes
+ */
+enum cib_op_attr {
+    cib_op_attr_none       = 0,         //!< No special attributes
+    cib_op_attr_modifies   = (1 << 1),  //!< Modifies CIB
+    cib_op_attr_privileged = (1 << 2),  //!< Requires privileges
+};
+
 typedef struct cib_operation_s {
     const char *operation;
-    gboolean modifies_cib;
-    gboolean needs_privileges;
+    enum cib_op_attr flags;
```

Comment:
uint32_t

Reply:
Ah yep. I can't remember *why* we prefer explicit types like `uint32_t` for a struct member... if it were just to ensure a strict size for the struct, we'd never use `int` either.

Reply:
Because the member does not hold an enum value, but a bitmask of enum values. It's unsigned since bitmasks should always be, and specifying the size ensures we have enough for the number of bits defined

Reply:
Ah, that's right.